scripts/GetRandomInt/GetRandomInt.gml

```diff
@@ -0,0 +1,8 @@
+/// @description GetRandomInt(min,max) Gets a random integer in range inclusively defined by parameters
+/// @arg min minimum integer
+/// @arg max maximum integer
+var minInt = argument[0];
+var maxInt = argument[1];
+
+var rnd = random_range(minInt, maxInt+1);
```

Comment:
This is giving the same random numbers each time you run the game. Do we want to call the `randomize()` function somewhere at the start of the game? Or we could keep the random seed the same every game for now for debugging purposes and change it down the road?